pkgs/misc/screensavers/electricsheep/default.nix

```diff
@@ -1,22 +1,38 @@
-{stdenv, fetchurl, pkgconfig, expat, zlib, libpng, libjpeg, xorg}:
+{stdenv, fetchgit, pkgs}:
 
 stdenv.mkDerivation rec {
-  name = "electricsheep-2.6.8";
+  name = "electricsheep";
+  version = "2.7b33";
   
-  src = fetchurl {
-    url = "http://electricsheep.org/${name}.tar.gz";
-    sha256 = "1flqcqfs75wg74hr5w85n6w8b26l4qrpwzi7fzylnry67yzf94y5";
+  src = fetchgit {
+    url = "https://github.com/scottdraves/${name}.git";
+    rev = "598d93d90573b69559463567540aac8bc8a5b3f3";
+    sha256 = "1zcn6q0dl0ip85b8b4kisc5lqjb1cs0hpzlx4l5995l6mhq9kxis";
   };
 
-  buildInputs = [pkgconfig expat zlib libpng libjpeg xorg.xlibsWrapper xorg.libXv];
+  buildInputs = with pkgs; [
+    autoconf automake libtool wxGTK30 libav lua5_1 pkgconfig curl libpng xorg.libXrender
```

Comment:
Instead of adding `autoconf` and `automake` and calling `./autogen.sh` you can just use the `autoreconfHook`.